karma.conf.js

```diff
@@ -3,6 +3,8 @@
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 const { TypedCssModulesPlugin } = require('typed-css-modules-webpack-plugin');
 
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
are you sure this is necessary?  I didn't have any trouble with chrome headless on GHA.  I'd prefer to avoid adding another dependency if possible.